### DISCUSSION TITLE
X10 gift box

### DISCUSSION THREAD

**Discussion by KevoHack** [HeyuX10Automation/heyu#70] [General]
Hello,

I am uninstalling my entire X10 home automation due to circumstances beyond my control.

Inspired by Bromaynard https://github.com/Bromaynard, I would like to gift everything to a fan of X10, shipping at my expense.

Also I would like to thank the Heyu developers https://www.heyu.org/heyu_credits.html and most especially Janusz Krzysztofik https://github.com/jkrzyszt whose ports to Linux made it possible to run Heyu on Raspberry pi

The box contains:

2x CM11A 2-Way Computer Interface
3x RR501 Transceiver Module
3x CR12A Remote Control
3x AM466 Appliance Module
2x LM465 Lamp Module
2x XPFM Fixture Module
2x XPDI3 Decorator Master Wall Dimmer Module
2x WS467 Wall Switch Module
2x WS12A Decorator Master Wall Dimmer Module
1x XPCR? PRO Wired-In Coupler Repeater Amplifier
12+ MS16A Indoor/Outdoor Motion Sensor (some NIB, some weatherbeaten, all working)

~Kevo

**Comment by arnieSkyNet** [comment]
Very nice thing to do and very nice mentions.

Brought to mind the originator of the heyu software Charles W. Sullivan, though I'm pretty much thinking about him each day when Alexa controls the hayu stuff via HA bridge

**Reply by KevoHack** [reply]
ArnieSkyhNet,

Thank you for your kindly response.  We heyu'users owe much to Dr. Sullivan and the half-dozen others that made heyu work heyugreat!

Its wonderful of you to say I am doing a nice thing. Actually I need to gift these devices because I know 100% they work with heyu on Rasbian, Redhat, Fedora, and a bunch of others.  It would be a heartbreak to send this box of hundreds of dollars of working stuff to a landfill.  I don't even care if the recipient wants one item and resells the rest.

Your Alexa HA bridge, is it one way (can turn on lamps) or two way (also can get X10 motion sensor reports)?

~Kevo

**Reply by arnieSkyNet** [reply]
I don't use HA Bridge to get feed back, some seem to do things.

A lot seem to go over to home Assistant, I had little success on migration to it.

I've been buying U.K. stuff via eBay, I'm all x10 sorted now. 

I bought a USA version computer controller and another U.K. device that a YouTuber who loathes internet iot is going to try a conversion and a repair of my 1st computer controller where the eprom has failed (worn out) plus he might try repairing some blown x10 modules that I sent him.

**Reply by CPUGPU** [reply]
I'm lucky as I've never had an X-10 module fail in 30+ years.  I'm still using an original thick brown BSR-branded appliance module!

**Reply by arnieSkyNet** [reply]
My own fault, the modules I overloaded plugging in appliances drawing too much current and the CM11A 2-Way Computer Interface I was uploading once a day, to the EEPROM a new schedule every day for about two years to implement a self start system for the flat😂

**Comment by CPUGPU** [comment]
Thank you for this offer.  I'm a sucker for X-10.  Still use it extensively in my house.  Can you post direct contact info you don't mind being public?  I'd like to contact you directly about these.

**Reply by KevoHack** [reply]
Please see if a Githup direct message (DM) is possible/works, I believe I have my Githup setup with my proper email address for for this purpose.

**Reply by KevoHack** [reply]
Also being as I'm paying postage we don't have to fuss with credit card security.  I will of course send via Fed Ex "Next Day or Week or Even Month--When We Get Around to It".

**Reply by CPUGPU** [reply]
Github stopped DMs more than a decade ago.  But I've figured out another way.  Check your github mailbox or notifications.  

If we make this happen, I'm going to donate a nice amount to a project or charity of your choice, or my choice if you decline.  I think this gift is worth paying forward!

**Reply by KevoHack** [reply]
> Github stopped DMs more than a decade ago. 

Serves me right for using a 12 year old Stack Overflow for instructions.

> But I've figured out another way. Check your github mailbox or notifications.

Worked, thank you, please check your corresponding inbox.

> If we make this happen, I'm going to donate a nice amount to a project or charity of your choice, or my choice if you decline. I think this gift is worth paying forward!

https://github.com/home-assistant is excellent, even supports X-10 (but one-way, I need two-way, and clunky via containerized version)

https://github.com/zwave-js/zwave-js-ui is excellent for Z-Wave (our home automation successor to X-10).

For anyone else reading, Home Assistant is much like X-10 w/ HeyU in that you run it at home, on your own system(s), no cloud intrusion is required.

**Reply by CPUGPU** [reply]
Both of those are worthy projects.  I support HA already, but happy to do more!  

I use HA (minimally) with X10 integration (which requires Heyu) for basic timed control.  I've been meaning to expand it.  Now I can. 

Thank you!

**Comment by KevoHack** [comment]
In conclusion, the gift box has been delivered to a genuine X10 enthusiast.  I am very glad someone else can make use of these wonderful if slightly obsolete gizmos and grateful he made donations to open source projects for the box.  

The many heyu contributors have my gratitude and have inspired me to contribute code to other open source projects.  

So long, and thanks for all the fish!

**Comment by CPUGPU** [comment]
Can confirm:  KevoHack _very generously_ delivered a big box full of X10 devices to me.   To help pay it forward, I've made modest gifts to Z-Wave JS UI and The Internet Archive. (The Archive is where a lot of old X10 sites live now...)

Thank you KevoHack!
